docs/introduction.md

```diff
@@ -85,3 +85,71 @@ Available download clients and actions
 
 Autobrr was developed with resource consumption in mind. The software uses API calls to reduce unnecessary downloads of .torrent files from sites like BTN, RED, PTP, and GGn. On other sites, it will download the .torrent only if the information is not present in the announce message.
+
+## Quick Start
+
+The following considers a common use case, feeding IRC announcements for a private
+tracker to a Servarr instance, as a way to illustrate how the components of autobrr fit
+together and what is required to get up and running.  It's not the only use case and
+other use cases may require more RTFM'ing in [the configuration docs](./configuration).
+
+1. [Install autobrr](./installation/linux).
+
+	Proceed once the web UI is accessible.
+
+2. Create a user for yourself.
+
+	Most easily done through the initial "wizzard" in the web UI when autobrr is first
+	started.
+
+3. [Register a nick on your indexer's IRC
+   network](https://autobrr.com/configuration/irc#registering-with-nickserv).
+
+4. [Group a "bot" nick with your real nick](./configuration/irc#grouping-nicks)
+
+5. Add an [indexer](./configuration/indexers).
+
+6. Add a [Servarr instance](./configuration/download-clients#sonarr)
+
+	**NOTE**: In the context of autobrr, Servarr instances are considered download
+	clients.
+
+7. Add a [filter](./filters).
+
+	To feed all IRC announcements to a Servarr instance to let it decide what, if
+	anything, to do with the release, just add a filter with the indexer from #5
+	selected and leave the rest blank.  Don't forget to enable the filter.
+
+	**NOTE**: Autobrr does nothing with received IRC announcements without at least one
+	filter applied to at least one indexer.
+
+8. Add a [filter action](./filters/actions).
+
+	Add an action of the matching Servarr type (e.g. Sonarr type action for a Sonarr
+	instance), select the "download client" that corresponds to that type from #6, give
+	it a name and save.
+
+	**NOTE**: Autobrr does nothing with received IRC announcements without at least one
+	action enabled in at least one filter.
+
+9. Double check the resulting settings so far.
+
+	Review all the indexer, download client, and IRC network settings in the autobrr web
+	UI and correct any errors and omissions.  Ensure that everything but the IRC network
+	is enabled.
+
+10. Enable the IRC network.
+
+	**NOTE**: The network will display as `unhealthy` and `network unhealthy` messages
+	appear in the logs until authentication has succeeded and autobrr has successfully
+	joined the announcements channel.  So you may safely ignore these messages until the
+	logs show further information about connecting, authenticating and joining the
+	channel.
+
+Now you can monitor the logs for announcements from IRC, pushes to the Servarr instance,
+and details for what, if anything, Servarr did with the release:
+
```

Comment:
I'd probably use fenced code blocks for the log output here.

```text
INFO Matched 'Foo Series S01E01 1080p WEB h264-BarGrp' (All) for foo-indexer
DEBUG release.store: &{ID:12 FilterStatus:FILTER_APPROVED Rejections:[] Indexer:foo-indexer FilterName:All Protocol:torrent Implementation:IRC Timestamp:1970-01-01 00:00:00.000000000 -0000 UTC m=+0000.000000000 GroupID: TorrentID:######### TorrentURL:https://www.example.com/Foo+Series+S01E01+1080p+WEB+h264-BarGrp.torrent TorrentTmpFile: TorrentDataRawBytes:[] TorrentHash: TorrentName:Foo Series S01E01 1080p WEB h264-BarGrp Size:0 Title:Foo Series Category:TV :: Episodes HD Categories:[] Season:1 Episode:1 Year:0 Resolution:1080p Source:WEB Codec:[H.264] Container: HDR:[] Audio:[] AudioChannels: Group:BarGrp Region: Language: Proper:false Repack:false Website: Artists: Type: LogScore:0 IsScene:false Origin: Tags:[] ReleaseTags: Freeleech:false FreeleechPercent:0 Bonus:[] Uploader:Anonymous PreTime: Other:[] RawCookie: AdditionalSizeCheckRequired:false FilterID:1 Filter:0x########## ActionStatus:[]}
DEBUG sonarr: release push rejected: Foo Series S01E01 1080p WEB h264-BarGrp, indexer foo-indexer to http://localhost:8989 reasons: '[Unknown Series]'
DEBUG release rejected: Unknown Series
```

Reply:
Apologies, I'm much more of a reStructuredText devotee than I am knowledgeable in markdown.  Can you elaborate/educate?

Reply:
Like this: https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

Reply:
Ah, will do.  Why do you prefer this so I can learn?

Reply:
Hmm, that renders all lines as one line when I use the CLI renderer locally:

    $ markdown --version

    This is Markdown, version 1.0.1.
    Copyright 2004 John Gruber
    http://daringfireball.net/projects/markdown/

LMK if you still prefer the fenced version and I'll push the change.

Reply:
It should not render them as one line if you use 3 backticks before and after the first and last line.
One backtick will put all in one line though.

I guess its just more appealing visually in the document. It looks messy if the log output is just a regular paragraph. That's just my opinion though 😄

Reply:
I like that argument for the developer/writer experience of reading the souce.  I pushed the change.  This version of it still renders as one line in my Chrome from that version of the CLI markdown tool, just FYI.  Odd:

    <p><code>
    INFO Matched 'Foo Series S01E01 1080p WEB h264-BarGrp' (All) for foo-indexer
    DEBUG release.store: &amp;{ID:12 FilterStatus:FILTER_APPROVED Rejections:[] Indexer:foo-indexer FilterName:All Protocol:torrent Implementation:IRC Timestamp:1970-01-01 00:00:00.000000000 -0000 UTC m=+0000.000000000 GroupID: TorrentID:######### TorrentURL:https://www.example.com/Foo+Series+S01E01+1080p+WEB+h264-BarGrp.torrent TorrentTmpFile: TorrentDataRawBytes:[] TorrentHash: TorrentName:Foo Series S01E01 1080p WEB h264-BarGrp Size:0 Title:Foo Series Category:TV :: Episodes HD Categories:[] Season:1 Episode:1 Year:0 Resolution:1080p Source:WEB Codec:[H.264] Container: HDR:[] Audio:[] AudioChannels: Group:BarGrp Region: Language: Proper:false Repack:false Website: Artists: Type: LogScore:0 IsScene:false Origin: Tags:[] ReleaseTags: Freeleech:false FreeleechPercent:0 Bonus:[] Uploader:Anonymous PreTime: Other:[] RawCookie: AdditionalSizeCheckRequired:false FilterID:1 Filter:0x########## ActionStatus:[]}
    DEBUG sonarr: release push rejected: Foo Series S01E01 1080p WEB h264-BarGrp, indexer foo-indexer to http://localhost:8989 reasons: '[Unknown Series]'
    DEBUG release rejected: Unknown Series
    </code></p>